lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -341,6 +341,7 @@
             <?php if (isset($reportback_copy)): ?>
               <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
             <?php endif; ?>
+            <h3 class="inline-alt-text-color"><?php print t('JOIN OTHERS MAKING A DIFFERENCE'); ?></h3>
```

Comment:
Tag use looks fine, but ideally it wouldn't be written all out in CAPS and instead can use CSS for that. But I don't believe we have a class to specifically capitalized a single (normally not capitalized) item. Can you add a `@TODO` for the fenders here to look into maybe adding a `.-yelling` (or more appropriately named) class?